services/ui-src/src/components/forms/Form.tsx

```diff
@@ -70,7 +71,10 @@ export const Form = ({
     <FormProvider {...form}>
       <form
         id={id}
-        onSubmit={form.handleSubmit(onSubmit as any, onErrorHandler)}
+        onSubmit={form.handleSubmit(
+          onSubmit as any,
+          onError || (onErrorHandler as any)
```

Comment:
Nit: Could tighten up these types. Though I know they are the internal RHF types, and the error handler one will probably get mad.